### PR TITLE
increase pandas timeout further

### PR DESCRIPTION
**Pull request by salsal97**

### CODE REVIEW

File: solutions/pandas_tests/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ PYTEST_INI_FILE = pytest.ini
 PYTEST_OPTS = -s -q --no-header --skip-slow --skip-db --skip-network
 TEST_ROOT = /usr/local/lib/python3.9/site-packages/
 TEST = /usr/local/lib/python3.9/site-packages/pandas/tests/io/test_fsspec.py 
-TIMEOUT = 1800
+TIMEOUT = 3000
```
Comment (jxyang) [inline solutions/pandas_tests/Makefile:18]: Do we know which test case(s) are taking too long?

Comment (salsal97) [reply solutions/pandas_tests/Makefile:18]: The increase is explained by the fact that we are building + running in the "make tests" target now..

Comment (jxyang) [reply solutions/pandas_tests/Makefile:18]: How long does it take to build pandas solution test?

Comment (salsal97) [reply solutions/pandas_tests/Makefile:18]: I think my previous assessment was incorrect, my guess is that it takes a long time to run on coffelake. ~36 minutes currently on coffeelake whereas it always finishes within 30 minutes on icelake

Comment (salsal97) [reply solutions/pandas_tests/Makefile:18]: My previous timeout was determined using icelake

**Review by jxyang — APPROVED** [review]

**Pull request merged**